taxi/views.py

```diff
@@ -28,6 +30,22 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required()
+def car_driver_assignment(
+        request: HttpRequest,
+        pk: int,
+        driver_id: int
+) -> HttpResponse:
+    car = get_object_or_404(Car, pk=pk)
+    driver = get_object_or_404(Driver, pk=driver_id)
```

Comment:
since we got login_required decorator, do we need get_object_or_404?, also checklist )